packages/utils/src/modify-config-helper.ts

```diff
@@ -103,9 +105,10 @@ export function modifyHelperUtil(
                 const confPath = path.resolve(process.cwd(), '.yo-rc.json');
                 configModule = require(confPath);
             } catch (err) {
-                console.error(red('\nCould not find a yeoman configuration file (.yo-rc.json).\n'));
-                console.error(
-                    red("\nPlease make sure to use 'this.config.set('configuration', this.configuration);' at the end of the generator.\n"),
+                logger.raw();
+                logger.error('Could not find a yeoman configuration file (.yo-rc.json).\n\n');
```

Comment:
Let's put `\n` here, otherwise we will have problems with colors

Reply:
Same for other places

Reply:
```js
logger.error('\nCould not find a yeoman configuration file (.yo-rc.json).\n\n');
```
This would result in:-
```bash
[webpack-cli]
Could not find a yeoman configuration file (.yo-rc.json).
```

What we intend is to have the `[webpack-cli]` prefix followed by the respective message in the same line right?